Geometry/HGCalCommonData/src/HGCalGeomParameters.cc

```diff
@@ -690,7 +690,8 @@ void HGCalGeomParameters::loadSpecParsHexagon8(const DDFilteredView& fv,
 				<< php.cellThickness_[1] << ":" 
 				<< php.cellThickness_[2];
 #endif
-  php.radius100to200_ = DDVectorGetter::get("Radius100to200");
+  int nmin = 5;
```

Comment:
does `nmin` need to be a reused variable instead of just being a value given to the function directly?